packages/layout/src/ChartPanel.ts

```diff
@@ -213,6 +317,8 @@ export class ChartPanel extends Border2 implements IHighlight {
         }
         element.style("box-shadow", this.highlight() ? `inset 0px 0px 0px ${this.highlightSize()}px ${this.highlightColor()}` : "none");
         super.update(domNode, element);
+
+        this.responsivePostUpdate(_responsiveMode, domNode, element);
```

Comment:
Minor: It might be cleaner to have 3 methods: 
* postUpdateTiny
* postUpdateSmall
* postUpdateRegular